sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -301,13 +301,14 @@ final class TypeSig private(
       meta: RapidsMeta[_, _, _],
       exprMeta: BaseExprMeta[_],
       name: String,
-      willNotWork: String => Unit): Unit = {
+      willNotWork: String => Unit,
+      skipCheckUTC: Boolean = false): Unit = {
```

Comment:
Using this way, moving forward, we need to add ```skipCheckUTC``` for each supported operator.

How about adding a method like ```isTimezoneSupported``` for each meta? As ```BaseExprMeta```, it's default false. And overrides it once we support that expr with timezone.

It seems slightly better.

Reply:
Solution changed, this comment is invalid now.